### ISSUE TITLE
(React) Infinite Type Loop is not infinite. It stops.

### ISSUE THREAD

**Issue by Joroze** [mattboldt/typed.js#303]
### Description

I used Typed.js with my React app within one of my components. Based on the documentation that is provided, I believe I set up my Typed.js config right. Typed.js is expected to loop forever with random lines given. For some reason, it breaks/stops after a few loops.

### Steps to Reproduce
The steps is shown in the code below. 

**Expected behavior:** Typed.js is expected to keep typing continuously.

**Actual behavior:** Typed.js stops typing after a few loops

**Reproduces how often:** Reproduces basically every time, after a few loops.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

```
import React, {
    Component
} from 'react';
import {
    connect
} from 'react-redux';
import {
    Container,
    Divider,
    Header,
    Icon
} from 'semantic-ui-react';
import Typed from 'typed.js';

import MySplitButton from './buttons/MySplitButton';
import {
    performUserProfileRequest
} from './MyJumbotron.duck.js';

class MyJumbotron extends Component {

    componentDidMount() {
        const {
            strings
        } = this.props;

        this.props.loadUserProfile("joroze")

        const options = {
            strings: strings,
            typeSpeed: 100,
            backSpeed: 50,
            backDelay: 2000,
            showCursor: false,
            shuffle: true,
            loop: true,
            loopCount: Infinity
        };

        // this.el refers to the <span> in the render() method
        this.typed = new Typed(this.el, options);
    }

    componentWillUnmount() {
        // to prevent memory leaks
        this.typed.destroy();
    }

    render() {
        return (
            <div className='jumbotron'>
                <Container textAlign='center'>
                    <Header className='white' icon>
                        <Icon name='at' />
                        <Header.Content>
                            Jordan Rosenberg
                            <Header.Subheader className='white'>
                                <span ref={(el) => { this.el = el; }}/>
                            </Header.Subheader>
                        </Header.Content>
                    </Header>

                    <Divider horizontal inverted>
                        <MySplitButton></MySplitButton>
                    </Divider>

                </Container>
                </div>
        );
    }
}

function mapDispatchToProps(dispatch) {
    return {
        loadUserProfile: (username) => dispatch(performUserProfileRequest(username))
    };
}

function mapStateToProps(state) {
    return {
        user: state.user
    };
}

export default connect(mapStateToProps, mapDispatchToProps)(MyJumbotron);

```

**Comment by mattboldt** [comment]
Yep, I've just reproduced this in the react demo. Are there any console errors showing up for you? 

I'll dig in further soon, but for the time being, feel free to submit a pull request

**Comment by Joroze** [comment]
Hey @mattboldt, I believe I fixed the problem :)

It's now working correctly on my website http://joroze.com/ (hosted on GitHub pages).

I noticed that Typed.js completely removes the whole string (and "Types" a new one) through every loop. With that said, the span has no content and the empty line is removed from the DOM in a way.

My solution was setting the `<span>`'s CSS display to inline-block. It seems to be working now. :)
